bin/install.sh

```diff
@@ -102,9 +103,8 @@ if [[ "$OS" = "Linux" && $(cat /proc/1/cgroup) == *cpu:/docker/* ]]; then
   # Java Install
   echo -e "\033[1;36mStarting Java install...\033[0m"
 
-  sudo add-apt-repository -y ppa:openjdk-r/ppa
   sudo apt-get update
-  sudo apt-get install openjdk-8-jdk libgfortran3 -y
+  sudo apt-get install openjdk-7-jdk libgfortran3 -y
```

Comment:
Yes, but it's not in this merge. We had all of the SF 0.9.6 running on j7, I'll fix this to say j8 for now